data/brands/shop/clothes.json

```
@@ -14,6 +14,17 @@
     }
   },
   "items": [
+  {
+      "displayName": "Earthbound Trading Company",
+      "locationSet": {"include": ["001"]},
```

```suggestion
      "locationSet": {"include": ["us"]},
```
per https://www.earthboundtrading.com/locations/